data-structures/src/main/java/com/baeldung/list/CircularLinkedList.java

```diff
@@ -0,0 +1,102 @@
+package com.baeldung.list;
+
+public class CircularLinkedList {
+
+    Node head = null;
```

Comment:
Could we make these members private? I don't think they need to be available outside the class right?